drivers/sx127x/sx127x.c

```diff
@@ -252,11 +258,11 @@ static int _init_peripherals(sx127x_t *dev)
     res = spi_init_cs(dev->params.spi, dev->params.nss_pin);
 
     if (res != SPI_OK) {
-        DEBUG("sx127x: error initializing SPI_%i device (code %i)\n",
+        DEBUG("[sx127x] error initializing SPI_%i device (code %i)\n",
                   dev->params.spi, res);
         return 0;
     }
 
-    DEBUG("sx127x: peripherals initialized with success\n");
+    DEBUG("[sx127x] peripherals initialized with success\n");
```

Comment:
peripheral ?

Reply:
There are several peripheral initialized by this function: gpios and spi. But I admit that only the spi initialization return status is taken into account. I'll change that.

Reply:
refactored the error management in the drivers and test application